tools/helium.py

```diff
@@ -1469,6 +1470,9 @@ def parse_case_statement(self):
             conditions = []
             while True:
                 t = self.tokens[self.i]
+                if t is OTHERS:
```

Comment:
Same comments about the parsing here applies as in `parser.cpp`. I think doing it the way I suggest will be easier.